src/w3d/renderer/segline.cpp

```diff
@@ -73,7 +73,7 @@ void SegmentedLineClass::Set_Point_Location(unsigned int point_idx, const Vector
     Invalidate_Cached_Bounding_Volumes();
 }
 
-void SegmentedLineClass::Get_Point_Location(unsigned int point_idx, Vector3 &loc)
+void SegmentedLineClass::Get_Point_Location(int point_idx, Vector3 &loc)
 {
     if (point_idx < m_pointLocations.Count()) {
```

Comment:
`m_pointLocations` (SimpleDynVecClass<>) is using `int` for indexing. Therefore `unsigned int` in this class will issue compiler warning when interacting with member class.